apps/console/src/extensions/configs/application.tsx

```diff
@@ -89,6 +89,12 @@ const featureConfig: FeatureConfigInterface = window[ "AppUtils" ].getConfig().u
  * @returns boolean
  */
 const isIdentityClaim = (claim: ExtendedClaimInterface | ExtendedExternalClaimInterface): boolean => {
+    const excludeIdentityClaims: boolean = window[ "AppUtils" ]?.getConfig()?.ui?.excludeIdentityClaims;
```

Comment:
Can't we pass this as an argument, so we can use the `useSelector` hook from component level and avoid accessing from window?

Reply:
`IsIdentityClaim` is only used inside the config file not in components. So, if we want to pass this using `useSelector` hook we need to pass it as chain from function to function. However, there is a one method it is not used anywhere, therefore I removed it https://github.com/wso2/identity-apps/pull/5350/commits/4cf97ff09386b2ca9eae4cea74b6131cba862bd5.
`IsIdentityClaim` is used only within the configuration file, not in components. Therefore, when passing it using the `useSelector` hook, it requires traversing through a chain of functions. However, there is one method that is not utilized anywhere; consequently, it has been removed. https://github.com/wso2/identity-apps/pull/5350/commits/4cf97ff09386b2ca9eae4cea74b6131cba862bd5